sample/ACPCoreSample/android/app/build.gradle

```diff
@@ -15,9 +15,14 @@ import com.android.build.OutputFile
  *   // the name of the generated asset file containing your JS bundle
  *   bundleAssetName: "index.android.bundle",
  *
- *   // the entry file for bundle generation
```

Comment:
Why so many changes for Android when only adding tvOS?

Reply:
@nporter-adbe 
I will work on fixing the ci. Since the new sample app is working with both iOS and tvOS, Do you think we can keep one app rather than two separate apps for tvOS and iOS?

Reply:
See my comment below, I think we should keep the original sample app, then add a tvOS specific app. This will reduce the work you'll need to do on CI.